src/components/pagination/_pagination.scss

```diff
@@ -315,10 +314,7 @@ $css--helpers: true;
 
     &:disabled:hover {
       cursor: default;
-
-      .#{$prefix}--pagination__button-icon {
```

Comment:
@tw15egan An update since your review: I just committed something I forgot 😅
About to update the changelog in the desc as well

Reply:
Weird, I'm seeing hover effects on the disabled version (both on v9 and v10). Disabled styles may need some more work. We can create another issue if that's easier

Reply:
In that case would you rather I revert this update, or just keep as-is? (I'm thinking the former, since `.#{$prefix}--pagination__button-icon` isn't being applied in the React component anymore?)

Reply:
Should we update the React component instead to apply that class? Would that fix the issue?

Reply:
I thought about doing that at first, but I wasn't sure if the React version was meant to take a different direction... here are some screenshots of the rendered button between Carbon X and current Carbon:

**Carbon X**
![screen shot 2019-02-20 at 11 51 58 am](https://user-images.githubusercontent.com/9057921/53112964-1468d000-3506-11e9-8fa2-8ad9ce32ba85.png)

**Carbon v9**
![screen shot 2019-02-20 at 11 52 55 am](https://user-images.githubusercontent.com/9057921/53112965-1468d000-3506-11e9-85cd-677414da819c.png)

There difference in code is:
```javascript
{componentsX ? (
  <CaretLeft24 />
) : (
  <Icon
    className={`${prefix}--pagination__button-icon`}
    icon={iconChevronLeft}
    description={backwardText}
  />
)}
```

So the new `<CaretLeft24 />` already comes equip with a lot of required attributes 🤔 Just not the class.

In this case... might be easy enough to just do like `<CaretLeft24 className={`${prefix}--pagination__button-icon`} />` in the React library 😅 We'd still need a style rule to apply `fill: $ui-05` to `${prefix}--pagination__button-icon` here, however.

Reply:
Ahhhhh I see. Yeah, let's go with your solution and add the className in the React version, and we can make the updates here in the meantime